packages/react-vapor/src/components/collapsible/CollapsibleContainerConnected.tsx

```diff
@@ -72,7 +70,10 @@ export const CollapsibleContainerDisconnected: React.FunctionComponent<
                 </div>
             }
             expandedOnMount={expandedOnMount}
-            headerClasses={classNames(styles.header, expanded ? styles.expanded : styles.collapsed)}
+            headerClasses={classNames(
+                'collapsible-container header' +
+                    (expanded ? 'collapsible-container expanded' : 'collapsible-container collapsed')

```

Comment:
```tsx
classNames(`collapsible-container header ${expanded ? 'collapsible-container expanded' : 'collapsible-container collapsed'}`)
```

Reply:
I may be wrong but I think the current way using `+` would join `header` and `collapsible-container` together?

Reply:
Not using the + is breaking the style so i'll keep it for now :sweat_smile: